### ISSUE TITLE
Does not continue with transfering a list of playlist, if one list has missing songs

### ISSUE THREAD

**Issue by markimark** [mauimauer/portify#17]
The title says it all. If I mark a few lists for transfer and one of them has missing songs, the app stops working, just showing the message of the number of songs, transfered and missing ones and freezes at that point. You then can go "back" chose another list, hopefully one that does not have any missing songs.

**Comment by mauimauer** [comment]
Yep...that issue has come up in the past, but unfortunately having missing songs does not always yield in the situation you just described. I'll check and see what I can do...but this after all an unsupported api that can change any minute.
